value-processor/src/org/immutables/value/processor/Immutables.generator

```diff
@@ -1253,6 +1253,7 @@ public interface BuildFinal[type.generics] {
   @SuppressWarnings("unchecked")
 [/if]
   private void from(Object object) {
+    [dynamicFromModifiableCheck type 'object' '']
```

Comment:
`return ;` with extra space would be a little ugly here, maybe add some conditions inside `dynamicFromModifiableCheck`, like `return[if returnValue] [returnValue][/if];`